modules/ivc_champva/app/form_mappings/vha_10_7959c.json.erb

```diff
@@ -27,32 +27,32 @@
   "form1[0].#subform[0].NameofInsurance-1[0]":                                "<%= form.data.dig('applicant_primary_provider') %>",
   "form1[0].#subform[0].Date-NameInsurance-1[0]":                             "<%= form.data.dig('applicant_primary_effective_date') %>",
   "form1[0].#subform[0].Date-TermnNameInsurance-1[0]":                        "<%= form.data.dig('applicant_primary_expiration_date') %>",
-  "form1[0].#subform[0].#area[2].CheckBox-HMO-1[0]":                          "<%= form.data['applicant_primary_insurance_type']&.key?('hmo') ? 1 : 0 %>",
-  "form1[0].#subform[0].#area[2].CheckBox-PPO-1[0]":                          "<%= form.data['applicant_primary_insurance_type']&.key?('ppo') ? 1 : 0 %>",
-  "form1[0].#subform[0].#area[2].CheckBox1-MedicaidSA-1[0]":                  "<%= form.data['applicant_primary_insurance_type']&.key?('medicaid') ? 1 : 0 %>",
-  "form1[0].#subform[0].#area[2].CheckBox-RxDiscount-1[0]":                   "<%= form.data['applicant_primary_insurance_type']&.key?('rx_discount') ? 1 : 0 %>",
-  "form1[0].#subform[0].#area[2].#area[3].CheckBox-Medigap-1[0]":             "<%= form.data['applicant_primary_insurance_type']&.key?('medigap') ? 1 : 0 %>",
+  "form1[0].#subform[0].#area[2].CheckBox-HMO-1[0]":                          "<%= form.data.dig('applicant_primary_insurance_type') == 'hmo' ? 1 : 0%>",
+  "form1[0].#subform[0].#area[2].CheckBox-PPO-1[0]":                          "<%= form.data.dig('applicant_primary_insurance_type') == 'ppo' ? 1 : 0%>",
+  "form1[0].#subform[0].#area[2].CheckBox1-MedicaidSA-1[0]":                  "<%= form.data.dig('applicant_primary_insurance_type') == 'medicaid' ? 1 : 0%>",
+  "form1[0].#subform[0].#area[2].CheckBox-RxDiscount-1[0]":                   "<%= form.data.dig('applicant_primary_insurance_type') == 'rx_discount' ? 1 : 0%>",
+  "form1[0].#subform[0].#area[2].#area[3].CheckBox-Medigap-1[0]":             "<%= form.data.dig('applicant_primary_insurance_type') == 'medigap' ? 1 : 0%>",
```

Comment:
We switched from a checkbox to a radio field for this data on the frontend, so had to update how we access it.